src/ServiceStack.Common/Script/ScriptConfig.cs

```diff
@@ -36,7 +37,8 @@ public static class ScriptConfig
         public static string DefaultErrorClassName { get; set; } = "alert alert-danger";
         public static bool AllowUnixPipeSyntax { get; set; } = true;
         public static bool AllowAssignmentExpressions { get; set; } = true;
-        
+        public static ParseRealNumber ParseRealNumber = numLiteral => numLiteral.ParseDouble();
```

Comment:
Couldn't define as Func<ReadOnlySpan<char>, object> because "ReadOnlySpan may not be used as a type argument for generic delegates"

https://stackoverflow.com/questions/53155438/why-readonlyspan-may-not-be-used-as-a-type-argument-for-generic-delegates-and-ge